integrations/access/slack/status.go

```diff
@@ -0,0 +1,51 @@
+package slack
+
+import (
+	"net/http"
+
+	"github.com/gravitational/teleport/api/types"
+)
+
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+func statusFromStatusCode(httpCode int) types.PluginStatus {
+	code := types.PluginStatusCode_OTHER_ERROR
+	switch code {
```

Comment:
This looks like a bug? Should be switching on httpCode, no?

Reply:
I think so, good catch! This was undertested in manual check since this is a last-resort basically. Will fix and add a test.